app/src/main/java/org/p2p/wallet/striga/wallet/repository/StrigaWalletRemoteRepository.kt

```diff
@@ -105,4 +111,27 @@ class StrigaWalletRemoteRepository(
             )
         }
     }
+
+    override suspend fun getUserWallet(): StrigaDataLayerResult<StrigaUserWallet?> {
```

Comment:
let's have this function return a non-null type and throw an exception if it can't find any wallets, that would be a pretty weird case, I reckon